cowprotocol/cowswap
# refactor: make TWAP warning less scary

# Summary
The TWAP warning was very technical and "scary" to a normal user. I've modified it such that it would be more understandable but a normal human being without going into the technicalities (fallbackhandler).

# To Test

1. Create a twap order on a safe that doesn't have cow's fallbackhandler. The message you see should look like this:
![grafik](https://github.com/user-attachments/assets/a0c3d983-d462-4a9e-975e-83d3b5be4a88)
